Volume/Analytics/Analytics.swift

```diff
@@ -8,36 +8,6 @@
 
 import AppDevAnalytics
 
-protocol Payload {
-    var eventName: String { get }
-    var parameters: [String: Any]? { get }
-}
-
-extension Payload {
-    var parameters: [String: Any]? {
-        nil
-    }
-}
```

Comment:
We needed this for announcements.  Why did you get rid of it?

Reply:
Parameters are already included in 
```
struct AnyEvent: Event {
    let name: String
    let parameters: [String: Any]?
}
```
All the stuff I deleted was code from the AppDevAnalytics pod, which should never have been in this file. The `Payload` structure and `logFirebase` function are outdated (from before my last PR).

All of this resolved the `Cannot find View in scope` (this could have also been solved by recreating the pods project + deleting derived data, so not entirely sure which did the trick).

Reply:
I terms of Announcements, that's why I included the `announcementPresent` enum in `VolumeEvent` and updated `MainView.swift`

Reply:
Ok sounds good!